packages/evolution-common/src/services/conditionals/__tests__/checkConditionals.test.ts

```diff
@@ -0,0 +1,352 @@
+            {
+                path: '_isNumber1',
+                comparisonOperator: '<=',
+                value: 2
+            }
+        ],
+        [true, null]
+    ],
+    [
+        'Path exist, but the value is smaller than or equal to 1',
+        [
+            {
+                path: '_isNumber1',
+                comparisonOperator: '<=',
+                value: 1
+            }
+        ],
+        [true, null]
+    ],
+    [
+        'Path exist, but the value is a number and the comparison operator is not valid',
+        [
+            {
+                path: '_isNumber1',
+                comparisonOperator: '?',
+                value: 1
+            }
+        ],
+        [false, null]
+    ],
+    [
+        'Path exist, but the value is an invalid object',
+        [
+            {
+                path: '_isNumber1',
+                comparisonOperator: '===',
+                value: { object: 'invalid' }
+            }
+        ],
+        [false, null]
+    ],
+    [
+        'Path exist, but the conditionals are multiple and the second one is false',
+        [
+            {
+                path: '_isNumber1',
+                comparisonOperator: '===',
+                value: 1
+            },
+            { path: '_isNumber1', comparisonOperator: '===', value: 0, logicalOperator: '&&' }
+        ],
+        [false, null]
+    ],
+    [
+        'Path exist, but the conditionals are multiple and the second one is false but the logical operator is OR',
+        [
+            {
+                path: '_isNumber1',
+                comparisonOperator: '===',
+                value: 1
+            },
+            { path: '_isNumber1', comparisonOperator: '===', value: 0, logicalOperator: '||' }
+        ],
+        [true, null]
+    ],
+    [
+        'Path exist, but the conditionals are multiple and the second one is false but the logical operator is OR inside parentheses',
+        [
+            {
+                path: '_isNumber1',
+                comparisonOperator: '===',
+                value: 1
+            },
+            { path: '_isNumber1', comparisonOperator: '===', value: 0, logicalOperator: '&&', parentheses: '(' },
+            { path: '_isNumber1', comparisonOperator: '===', value: 1, logicalOperator: '||', parentheses: ')' }
+        ],
+        [true, null]
+    ],
+    [
+        'Path exist, but the conditionals are multiple and the second one is false but the logical operator is AND inside parentheses',
+        [
+            {
+                path: '_isNumber1',
+                comparisonOperator: '===',
+                value: 1
+            },
+            { path: '_isNumber1', comparisonOperator: '===', value: 0, logicalOperator: '||', parentheses: '(' },
+            { path: '_isNumber1', comparisonOperator: '===', value: 1, logicalOperator: '&&', parentheses: ')' }
+        ],
+        [true, null]
+    ],
+    [
+        'Path exist, but the path have activePersonId inside',
```

Comment:
Tous les tests sont pour Path exists, il en faudrait un où le path does not exist (juste un suffirait)